management-interface.go

```diff
@@ -223,25 +222,77 @@ func SpeakerTestHandler(w http.ResponseWriter, r *http.Request) {
 	tmpl.ExecuteTemplate(w, "speaker-test.html", nil)
 }
 
+// exists returns whether the given file or directory exists
+func exists(path string) (bool, error) {
+	_, err := os.Stat(path)
+	if err == nil {
+		return true, nil
+	}
+	if os.IsNotExist(err) {
+		return false, nil
+	}
+	return true, err
```

Comment:
If os.Stat fails then it's highly unlikely that the file will be accessible to `play`. In this case it's probably fine to just have `exists` only return a bool and return false when `err != nil`.